portal/models/qb_timeline.py

```diff
@@ -412,7 +413,7 @@ def attempt_update(user_id, invalidate_existing):
 
         # acquire a multiprocessing lock to prevent multiple requests
         # from duplicating rows during this slow process
-        timeout = current_app.config.get("MULTIPROCESS_LOCK_TIMEOUT")
+        timeout = int(current_app.config.get("MULTIPROCESS_LOCK_TIMEOUT"))
```

Comment:
Is there a better place to cast this, so it will always be an `int` in `app.config.["MULTIPROCESS_LOCK_TIMEOUT"]`?

Reply:
I don't know of one.  The default in config.py IS an int - we ran into this when customized in `application.cfg`.  This is the only use of said configuration.  My attempt to recreate the problem fails.